stock_valuation_layer_accounting_date/models/stock_valuation_layer.py

```diff
@@ -21,6 +21,4 @@ def _compute_accounting_date(self):
             if account_move and account_move.state == "posted":
                 rec.accounting_date = account_move.date
                 continue
-            rec.accounting_date = fields.Datetime.context_timestamp(
-                self, rec.create_date
-            )
+            rec.accounting_date = rec.stock_move_id.accounting_date
```

Comment:
Shouldn't we still fall back on to `rec.create_date`, in case there is a SVL record **with no** `stock_move_id` and **with** `account_move_id` which is in draft state (i.e. user has temporarily changed the state to draft). Otherwise there will be a server errror I think.